### PR TITLE
Opower integration - Add ConEd

### PR DESCRIPTION
**Pull request by rct**
## Proposed change

Update Opower integration doc to include ConEd MFA configuration and ORU (opower) - https://github.com/home-assistant/core/pull/97878 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/97878 and https://github.com/home-assistant/core/pull/99230 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4625
- This PR fixes or closes issue: fixes #

Note: ConEd and ORU support is currently merged in opower but is disabled. Merging the MFA PR https://github.com/home-assistant/core/pull/97878 will enable it.  This doc PR should be merged after that PR to document the availability of the two new utilities.

(There is separate PR to add two new virtual integrations: https://github.com/home-assistant/core/pull/99230  This should be merged after https://github.com/home-assistant/core/pull/97878   There is a separate doc PR for the virtual integrations: https://github.com/home-assistant/home-assistant.io/pull/28726)

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/opower.markdown [hunk 1]
```diff
@@ -30,9 +31,35 @@ More than 175 utilities use Opower. Currently only the following ones are suppor
...
+
+If you have an existing TOTP set up, there are methods for exporting the account from your authenticator app and then using a tool to obtain the TOTP secret from the encoded string.
+
+Alternatively, you can create a new TOTP secret for your account and use the "no camera to scan" option to capture the TOTP secret. You may also want to capture the QR code by taking a picture, or using a QR code scanner. There are tools available that can decode the TOTP data from a QR code.
+
+**NOTE: At this time, ConEd only has a single TOTP set up per account. Therefore, it is important that you configure the same TOTP secret for ConEd access in both Opower and your authenticator app.**
+
+### Exelon subsidiaries
+
+When using Opower with any of the Exelon subsidiaries, such as such as BGE, ComEd, PECO,Pepco, etc., you need to actively disable two-factor authentication. Log onto the website, select **Don't use 2FA** and **Don't ask me again**. If you have already enabled 2FA, disable it.
```
Comment (tronikos) [inline source/_integrations/opower.markdown]: missing space after the comma

File: source/_integrations/opower.markdown [hunk 2]
```diff
@@ -17,8 +17,9 @@ ha_integration_type: integration
 
 The Opower integration allows you to get energy information from utilities that use [Opower](https://www.oracle.com/industries/utilities/opower-energy-efficiency/).
 
-More than 175 utilities use Opower. Currently only the following ones are supported:
+More than 175 utilities use Opower. Currently only the following utilities are supported by this integration:
 
+- Consolidated Edison (ConEd)
```
Comment (tronikos) [inline source/_integrations/opower.markdown]: Add "Orange & Rockland Utilities (ORU)" below this, similar to https://github.com/tronikos/opower/blob/main/README.md

**Review by tronikos — APPROVED** [review]

**Comment by tronikos** [comment]
Maintainers prefer small PRs. Split this to two separate PRs, each linked to one of the two parent PRs.

**Comment by rct** [comment]
> Maintainers prefer small PRs. Split this to two separate PRs, each linked to one of the two parent PRs.

Since the chances are so small and interrelated I thought it would be easier to follow if they were kept together.  

However, as you suggest, I've removed the commits for the new virtual integrations so this PR only covers the Opower page and will add a separate doc PR to accompany the virtual integration code change.

**Review by balloob — APPROVED** [review]

**Pull request merged**